rdt/transformers/__init__.py

```diff
@@ -87,10 +87,20 @@ def get_transformer_name(transformer):
 
 
 DEFAULT_TRANSFORMERS = {
-    'numerical': FloatFormatter(),
-    'categorical': FrequencyEncoder(),
-    'boolean': BinaryEncoder(),
-    'datetime': UnixTimestampEncoder(),
+    'numerical': FloatFormatter(
+        learn_rounding_scheme=False,
+        enforce_min_max_values=False,
+        missing_value_replacement='mean',
+        missing_value_generation='random',
+    ),
+    'categorical': LabelEncoder(add_noise=True),
+    'boolean': LabelEncoder(add_noise=True),
+    'datetime': UnixTimestampEncoder(
+        missing_value_replacement='mean',
+        missing_value_generation='random',
+    ),
+    'id': RegexGenerator(),
```

Comment:
I don't think we have the id sdtype in RDT. The `RegexGenerator` supports `text`, so this would crash. We can just excluse it or set the default for text instead here

Reply:
I see alright, done in [1df78ab](https://github.com/sdv-dev/RDT/pull/670/commits/1df78abe7f5a8222dceea2bf2e9ea9bf9520f3a1)